Source/d_main.c

```diff
@@ -1983,11 +1986,15 @@ void D_DoomMain(void)
   if ((p = M_CheckParm ("-timer")) && p < myargc-1 && deathmatch)
     {
       int time = atoi(myargv[p+1]);
+      timelimit = time;
```

Comment:
This already has an implementation in `p_spec.c:P_SpawnSpecials()`. Let's please use one or the other.

Reply:
Hmm, I wonder why this is in `P_SpawnSpecials()`?